curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.english.md

```diff
@@ -35,9 +35,13 @@ Declare a local variable <code>myVar</code> inside <code>myLocalScope</code>. Ru
 ```yml
 tests:
   - text: The code should not contain a global <code>myVar</code> variable.
-    testString: assert(typeof myVar === 'undefined');
+    testString: |
+      function declared(){
+        myVar;
+      }
+      assert.throws(declared, ReferenceError);
   - text: You should add a local <code>myVar</code> variable.
-    testString: assert(/function\s+myLocalScope\s*\(\s*\)\s*\{\s[\s\S]+\s*var\s*myVar\s*(\s*|=[\s\S]+)\s*;[\s\S]+}/.test(code));
+    testString: assert(/functionmyLocalScope\(\)\{.+(var|let|const)myVar.*;.+}/s.test(code.replace(/\s/g, '')));
```

Comment:
`testString: assert(/functionmyLocalScope\(\)\{.+(var|let|const)myVar.*;?.+}/s.test(code.replace(/\s/g, '')));`
Add `?` for optional `;`